src/Send/Context/SendContext.tsx

```diff
@@ -39,114 +76,77 @@ export const SendProvider: React.FC<SendProvider> = ({children, balances, wallet
 
 const initialState: SendState = {
   sendAll: false,
+  primaryTokenId: '',
```

Comment:
why did you add this?

Reply:
So we can use it in the reducer

Reply:
to reset the form without depending on anything external i.e.

Reply:
its duplicating state, if you can derive it, you should

Reply:
done